qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -924,14 +1227,25 @@ def parameters_table(
 
             keys.add(key)
 
-        for key in keys:
-            for value in self._params[key]:
-                value_dict = dataclasses.asdict(value)
-                value_dict["qubits"] = key.qubits
-                value_dict["parameter"] = key.parameter
-                value_dict["schedule"] = key.schedule
-                value_dict["date_time"] = value_dict["date_time"].strftime("%Y-%m-%d %H:%M:%S.%f%z")
-                data.append(value_dict)
+        # The following dictionary is used to keep track of the most recent parameter values.
```

Comment:
Should this comment move down two lines (or be dropped; it is confusing being next to data which is not a dict)?

Reply:
Done here: b8f12a1